sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -643,15 +644,14 @@ impl ItemRenderer for GLItemRenderer {
                 // trim is there to remove the \n
                 draw_line(&mut canvas, string[start..index].trim(), &mut y);
                 start = index;
-            }
-        } else {
-            let elide = text.overflow() == TextOverflow::elide;
-            'lines: for line in string.lines() {
-                if y + font_metrics.height() > max_height {
-                    break;
-                }
+            } else {
+                let index = string[start..].find('\n').map_or(string.len(), |i| start + i + 1);
```

Comment:
We used to call `string.lines()`, which also respects `\r\n`, but it's missing U+2028 and U+2029 (line and paragraph separator). I think we should replace all `\r\n`/`\n` and U+2029 with U+2028 and split on that.

That's slightly unrelated to this patch though, feel free to ignore :-)

Reply:
Note that the trim() on the next line will remove the \r as well.